pkgs/applications/video/mpv/default.nix

```
@@ -66,7 +66,7 @@
 , sdl2Support        ? true,           SDL2
 , sixelSupport       ? false,          libsixel
 , speexSupport       ? true,           speex
-, swiftSupport       ? false,          swift
+, swiftSupport       ? stdenv.isDarwin, swift
```

Might it be worth doing this for the time being and merging?

```suggestion
, swiftSupport       ? stdenv.isDarwin && stdenv.isAarch64, swift
```

---

Done in https://github.com/NixOS/nixpkgs/commit/8a28ae851dee5306de51352300fceb692bd9c389